src/languages/python/python.test.md

```diff
@@ -161,14 +161,14 @@ a-=a*a
 ```polygolf
 $a:0..oo <- 1;
 if ($a != 0) {
-    println_int $a;
+    $a <- 2;
 };
 $a <- 1;
 ```
 
 ```python
```

Comment:
Maybe we could support `python nohardcode` to specifically exclude the hardcode plugin.

Reply:
> Maybe we could support `python nohardcode` to specifically exclude the hardcode plugin.

Yes, I was thinking `no:pluginName`. `no:inlineVariables` would be useful too.